### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by StarMindz**
We created tests for all the Classes 
All tests were implemented using RSpec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ This is a useful tool for Librarians that helps to record what books are in the
  ### Key Features <a name="key-features"></a> 
   
  The tool allows librarians to 
+- Show persons.
```
Comment (DammyShittu) [inline README.md:49]: - [ ] Kindly add tests for the Person class. Remember to add tests for custom `add_rentals` and `can_use_services?` methods.

File: specs/book_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  context 'Object parameters' do
+    it 'returns the book title' do
+      @book.title.should eql 'Romeo and Juliet'
+    end
+
+    it 'returns the correct author' do
+      @book.author.should eql 'William Shakespeare'
+    end
+  end
+end
```
Comment (DammyShittu) [inline specs/book_spec.rb]: - [ ] Kindly add tests for the `add_rentals` method in the Book class

File: specs/student_spec.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    end
+
+    it 'returns the student age' do
+      @student.age.should eql 20
+    end
+
+    it 'returns the student name' do
+      @student.name.should eql 'Stanley'
+    end
+  end
```
Comment (DammyShittu) [inline specs/student_spec.rb]: - [ ] Kindly add tests for the `add_class` and `play_hooky` methods in the Student class

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @StarMindz and @Goruchie,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — COMMENTED** [review]
Hi @StarMindz and @Goruchie , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**